mars/oscar/backends/ray/backend.py

```diff
@@ -12,10 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ...backend import BaseActorBackend, register_backend
 from .driver import RayActorDriver
-from .context import RayActorContext
-
+from ..mars.context import MarsActorContext
```

Comment:
Maybe implement a RayActorContext, even though they are identical.

Reply:
MarsActorContext is pretty much code, about 150 lines. Do we need to duplicate this. I mean there won't be any difference for `RayActorContext`. All difference introduced by this PR are `actor pool` and `communication` level

Reply:
Moved `MarsActorContext` to `mars/oscar/backends/context.py`